hbase-testing-util/pom.xml

```diff
@@ -164,6 +164,10 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.codehaus.jackson</groupId>
```

Comment:
nit: a comment that we're including this here because Hadoop's minicluster would make it more maintainable.